### PR TITLE
Add `python-check-blanket-noqa` Hook.

### PR DESCRIPTION
**Pull request by adityasinghhh**

### CODE REVIEW

**Comment by adityasinghhh** [comment]
cc @asottile

File: .pre-commit-hooks.yaml [hunk 1]
```diff
@@ -1,3 +1,9 @@
+-   id: python-check-blanket-noqa
+    name: Check blanket noqa
+    description: 'Enforce that `noqa` annotation always occurs with specific codes to avoid blanket noqas'
```
Comment (asottile) [inline .pre-commit-hooks.yaml]: I'd say "Enforce that `noqa` annotation*s* always occur with specific codes"

Comment (adityasinghhh) [reply .pre-commit-hooks.yaml]: Fixed.

**Comment by asottile** [comment]
Thanks :tada:

**Pull request merged**